docs/content/intro/quickstart.md

```diff
@@ -0,0 +1,318 @@
+
+Let's now create a new file called `brigade.js` with the following content:
+
+```javascript
+const { events, Job } = require("brigadier");
+events.on("exec", () => {
+  var job = new Job("do-nothing", "alpine:3.8");
+  job.tasks = [
+    "echo Hello",
+    "echo World"
+  ];
+
+  job.run();
+});
+```
+
+When the `exec` event is triggered, Brigade will create a Build based on this brigade.js file. This Build will create a single job that will start an image based on `alpine:3.8` which will simply do a couple of echoes. The Kubernetes Pod that will be created to run this Build job will have a name starting with 'do-nothing'. As you can probably guess, a new Kubernetes Secret will also be created to store information about this Build.
+
+Moreover, you can check out [this tutorial](https://docs.brigade.sh/intro/) for more on creating scripts.
+
+### Running a Build
+
+To create and run a Brigade Build for the brigade.js file we wrote, we will use `brig`.
+
+```bash
+brig run brigadecore/empty-testbed -f brigade.js
+```
+
+With this command we are using the `brigade.js` file we just created. We could let Brigade use the `brigade.js` file in the `brigadecore/empty-testbed` repo, however this does not contain an `exec` event handler so nothing would happen. The usage of the custom `brigade.js` file we created lets us define a custom `exec` event handler.
+
+So, this command will trigger the `exec` event in the `brigade.js` file we created and show the detailed output, which will be similar to this:
+
+```
+Event created. Waiting for worker pod named "brigade-worker-01d0y7bcxs6ke0yayrx6nbvm39".
+Build: 01d0y7bcxs6ke0yayrx6nbvm39, Worker: brigade-worker-01d0y7bcxs6ke0yayrx6nbvm39
+prestart: no dependencies file found
+prestart: loading script from /etc/brigade/script
+[brigade] brigade-worker version: 1.0.0
+[brigade:k8s] Creating secret do-nothing-01d0y7bcxs6ke0yayrx6nbvm39
+[brigade:k8s] Creating pod do-nothing-01d0y7bcxs6ke0yayrx6nbvm39
+[brigade:k8s] Timeout set at 900000
+[brigade:k8s] Pod not yet scheduled
+[brigade:k8s] default/do-nothing-01d0y7bcxs6ke0yayrx6nbvm39 phase Pending
+[brigade:k8s] default/do-nothing-01d0y7bcxs6ke0yayrx6nbvm39 phase Succeeded
+done
+```
+
+As you can see, Brigade created a new pod for this Build (called `do-nothing-01d0y7bcxs6ke0yayrx6nbvm39`) that executed our job. Let's get its logs.
+
+```bash
+kubectl logs do-nothing-01d0y7bcxs6ke0yayrx6nbvm39
+```
+
+Output:
+```
+Hello
+World
+```
+
+Moreover, you can get the details for this Build from `brig`.
+
+```bash
+brig build list
+```
+
+Output:
+```
+ID                              TYPE    PROVIDER        PROJECT                                                         STATUS          AGE
+01d0y7bcxs6ke0yayrx6nbvm39      exec    brigade-cli     brigade-4897c99315be5d2a2403ea33bdcb24f8116dc69613d5917d879d5f  Succeeded       4m
+```
+
```

Comment:
Since this section covers using a git repo, could you add a note about how the VCS sidecar automatically clones the repo (and what to do for private repos), and what gateways to use for various git platforms?

Reply:
Added, let me know if you think more details are needed.